<?php

try {
    throw new RuntimeException();
} catch (Throwable $foo) {
}
